dev/drivers/scripts/stats/global_ens/jevs_global_ens_cmce_atmos_precip_stats.sh

```diff
@@ -32,7 +32,7 @@ evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
 export KEEPDATA=YES
 
 export vhr=00
-export COMIN=/lfs/h2/emc/vpppg/noscrub/${USER}/$NET/$evs_ver_2d
+export COMIN=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/$NET/$evs_ver_2d
```

Comment:
revert lines 14 and 35 to original definitions.

Reply:
Done.